spec/javascripts/NewService/formElements/__snapshots__/ServiceManualListItems.spec.tsx.snap

```diff
@@ -106,11 +116,22 @@ exports[`should render required form fields with errors 1`] = `
       type="text"
       value="new api"
     />
-    <p
-      className="inline-errors"
-    >
-      Only ASCII letters, numbers, dashes and underscores are allowed.
-    </p>
+    <HelperText>
+      <HelperTextItem>
+        <p
+          className="inline-errors"
+        >
+           
+          Only ASCII letters, numbers, dashes and underscores are allowed.
+           
```

Comment:
This line is not required anymore, right?

Reply:
but we don't update this manually

Reply:
we update jest using a command, no external changes applied

Reply:
and i think its needed it for different class inline-errors and inline hints

Reply:
Upated error message for system name so I think this is also fixed